bin/truffleruby

```diff
@@ -24,6 +24,30 @@ function mtime {
     fi
 }
 
+# Prefix any option in $@ with -J:, values are just copied over
+function prefix_options {
+    local result=()
+    for opt in "$@"
+    do
+        case $opt in
+            -*)
+                result+=("-J:${opt} ") ;;
```

Comment:
Does this function support spaces in options passed to GraalVM such as:
`-J-Dgraal.MethodFilter=a b` ?
I am also still wondering what the extra space in `result+=("$opt ")` does.